vernac/metasyntax.mli

```diff
@@ -24,6 +24,11 @@ val add_notation : locality_flag -> constr_expr ->
 
 val add_notation_extra_printing_rule : string -> string -> string -> unit
 
+(** Fine-grained activation and deactivation of notations *)
+
+val notation_activation :
```

Comment:
Sorry I am annoying but I still can't understand how this function works from the signature. Would you mind adding a comment?